config.go

```diff
@@ -41,8 +41,12 @@ type Config struct {
 
 	// YNAB related settings
 	YNAB struct {
+		// List of source for payee candidates
+		// Currently only name(=debtorName or creditorName) or unstructured are possible
+		PayeeSource []string `envconfig:"YNABBER_PAYEE_SOURCE" default:unstructured,name"`
+
```

Comment:
This setting should be moved to the Nordigen struct.

Reply:
Should the name be changed as well (i.e. NORDIGEN_PAYEE_SOURCE and NORDIGEN_PAYEE_STRIP)?
If so, wouldn't this break operation for current users?

Reply:
Yes let's rename them as well, we can keep the old name for PayeeStrip and simply map it to the new setting name.

Reply:
I moved these now but PayeeStrip environment value is still the same as old one. What did you mean by mapping? I.e. use environ NORDIGEN_PAYEE_STRIP if available, then YNABBER_PAYEE_STRIP for historical compatibility if available, or nothing.

Reply:
Lets just keep it as it is for now.